### ISSUE TITLE
Sort order for responsive classes

### ISSUE THREAD

**Issue by rushi** [avencera/rustywind#28]
I have the following broken sort order:
```html
<div className="px-0 md:flex md:flex-wrap lg:block md:px-8 pt-8 pb-0 lg:px-6">
```

Rustywind changes it to:
```html
<div className="px-0 pt-8 pb-0 md:flex md:flex-wrap lg:block md:px-8 lg:px-6">
```

According to [defaults.rs](https://github.com/avencera/rustywind/blob/master/src/defaults.rs), `block` has a higher priority than flex so shouldn't `lg:block` come before `md:flex`? If I assume `md` comes before `lg` then shouldn't `md:px-8` come before `lg:block`? 

I expect the correct order to be:
```html
<div className="px-0 pt-8 pb-0 md:flex md:flex-wrap md:px-8 lg:block lg:px-6">
```

I ask this because Adam Watham himself also suggests the [classes should be organized by screen size](https://twitter.com/adamwathan/status/1399473275709923332).

Thank you for rustywind, I'm finding it's sort order to be less buggier than headwind (I realize headwind uses rusty)

**Comment by praveenperera** [comment]
You're absolutely right the the sort order for responsive classes should follow the same logic. 

PR's are welcome, if you are able to. If not I will try and work on that next week. With the creator of Headwind being unresponsive I would like to make Rustywind better and maybe create a vscode extension using it. But I am pretty busy and I need some help.

**Comment by rushi** [comment]
I have a similar second issue

Original:
```html
<div className="grid md:grid-cols-3 sm:grid-cols-2 xs:grid-cols-1 2xl:grid-cols-4 gap-x-20 gap-y-10 mt-6">
```
Gets sorted as:
```html
<div className="grid gap-y-10 gap-x-20 mt-6 md:grid-cols-3 sm:grid-cols-2 xs:grid-cols-1 2xl:grid-cols-4">
```

Where `md:grid-cols-3` comes before `xs:grid-cols-1`

**Comment by rushi** [comment]
> PR's are welcome, if you are able to.

I would love to since my project's TW class order is a bit of a mess. But I have literally never worked in Rust. I looked in the source but couldn't find any code for processing responsive classes

**Comment by praveenperera** [comment]
That’s the problem right now. Responsive classes aren’t sorted. They are treated as non tailwind classes and sorted alphabetically. 

The general process i’m thinking is in `sort_classes_vec` would be to look in the custom classes vec and sort them if they are a responsive class.

**Comment by praveenperera** [comment]
This feature is complete and released in [v0.11.0](https://github.com/avencera/rustywind/releases/tag/v0.11.0)
